projects/1.21/assets/super-factory-manager/sfm/lang/zh_cn.json

```
@@ -0,0 +1,238 @@
+  "gui.sfm.manager.state": "状态：%s",
+  "gui.sfm.manager.state.invalid_program": "无效程序",
+  "gui.sfm.manager.state.no_disk": "未插入磁盘",
+  "gui.sfm.manager.state.no_program": "无程序",
+  "gui.sfm.manager.state.running": "运行中",
+  "gui.sfm.manager.status.fix": "正在修复问题！",
+  "gui.sfm.manager.status.loaded_clipboard": "已从剪贴板加载！",
+  "gui.sfm.manager.status.rebuild": "正在重建缓存！",
+  "gui.sfm.manager.status.reset": "已重置程序与标签！",
+  "gui.sfm.manager.status.saved_clipboard": "已保存到剪贴板！",
+  "gui.sfm.manager.tooltip.paste": "按 Ctrl + V 粘贴。",
+  "gui.sfm.manager.tooltip.reset": "清除磁盘所有数据。",
+  "gui.sfm.program_editor_config.intellisense": "智能提示",
+  "gui.sfm.program_editor_config.intellisense.advanced": "高级",
+  "gui.sfm.program_editor_config.intellisense.basic": "基础",
+  "gui.sfm.program_editor_config.intellisense.off": "关闭",
+  "gui.sfm.program_editor_config.line_numbers": "行号显示",
+  "gui.sfm.program_editor_config.title": "程序编辑器配置",
+  "gui.sfm.program_template_picker.warning1": "在下一界面点击“完成”将覆盖现有程序！",
+  "gui.sfm.program_template_picker.warning2": "按 <Esc> 取消操作。",
+  "gui.sfm.save_changes_confirm.message": "退出前是否保存更改？",
+  "gui.sfm.save_changes_confirm.no_button": "继续编辑",
+  "gui.sfm.save_changes_confirm.title": "保存更改",
+  "gui.sfm.save_changes_confirm.yes_button": "覆盖磁盘",
+  "gui.sfm.text_editor.config_button.tooltip": "打开编辑器配置",
+  "gui.sfm.text_editor.done_button.tooltip": "Shift + Enter 提交",
+  "gui.sfm.text_editor.title": "文本编辑器",
+  "gui.sfm.title.intellisense_pick_list": "智能提示选择列表",
+  "gui.sfm.title.labelgun": "标签枪",
+  "gui.sfm.title.program_template_picker": "程序模板选择器",
+  "item.sfm.disk": "工厂管理器程序磁盘",
+  "item.sfm.disk.tooltip.label_section.entry": " - %s: %d 个方块",
+  "item.sfm.disk.tooltip.label_section.header": "标签",
+  "item.sfm.form": "印刷模板",
+  "item.sfm.labelgun": "标签枪",
+  "item.sfm.labelgun.chat.pulled": "已从管理器拉取标签。",
+  "item.sfm.labelgun.chat.pushed": "已将标签推送到管理器。",
+  "item.sfm.labelgun.tooltip.1": "%s 以向工厂管理器推送标签。",
+  "item.sfm.labelgun.tooltip.2": "潜行时 %s 以从工厂管理器拉取标签。",
+  "item.sfm.labelgun.tooltip.3": "按住 %s 以将标签应用到线缆附近的同类方块。",
+  "item.sfm.labelgun.tooltip.4": "%s + %s 以从方块拾取标签。",
+  "item.sfm.labelgun.tooltip.5": "按 %s 以切换标签视图。",
+  "item.sfm.labelgun.with_label": "标签枪：\"%s\"",
+  "item.sfm.network_tool": "网络工具",
+  "item.sfm.network_tool.tooltip.1": "持有时透视显示线缆。",
+  "item.sfm.network_tool.tooltip.2": "右击方块面查看诊断信息。",
+  "item.sfm.network_tool.tooltip.3": "可能用不上，别忘了在物品栏按 %s 切换检查器。",
+  "item.sfm.network_tool.tooltip.4": "主手持有方块，副手持有此工具，右击线缆设置伪装。",
+  "item.sfm.network_tool.tooltip.5": "Ctrl + 点击 以连续伪装。",
+  "item.sfm.network_tool.tooltip.6": "Alt + 点击 以伪装网络中匹配的方块。",
+  "item.sfm.network_tool.tooltip.7": "Ctrl + Alt + 点击 以伪装整个网络。",
+  "item.sfm.xp_goop": "经验粘浆",
+  "item.sfm.xp_shard": "经验碎片",
+  "item_group.sfm.main": "超级工厂管理器",
+  "key.categories.sfm": "超级工厂管理器",
+  "key.sfm.container_inspector.activation_key": "切换容器检查器",
+  "key.sfm.item_inspector.activation_key": "（未完成）复制悬停物品到剪贴板",
```

| **审查级别** |
|:-:|
| ℹ️ 信息 / 不阻止合并  |

**原文: (WIP) Copy Hovered Item To Clipboard**

```suggestion
  "key.sfm.item_inspector.activation_key": "（未完成）复制悬停选中的物品到剪贴板",
```